library/src/main/java/bio/terra/landingzone/stairway/flight/CromwellStepsDefinitionProvider.java

```diff
@@ -50,6 +28,10 @@ public List<Pair<Step, RetryRule>> get(
                 parametersResolver),
             RetryRules.shortExponential()),
         Pair.of(new GetManagedResourceGroupInfo(armManagers), RetryRules.cloud()),
+        Pair.of(
+            new CreateNetworkSecurityGroupStep(
```

Comment:
So, all subnet will be associated with the same security group. Correct?

Reply:
Yes, for now since they will all have the same rules. If we need to add specific rules later, we can split it at that time.